include/seqan3/range/decorator/gap_decorator_anchor_set.hpp

```diff
@@ -650,7 +659,10 @@ class gap_decorator_anchor_set
      * \param[in] rhs The right-hand side gap decorator to compare.
      * \returns A boolean flag indicating (in)equality of the aligned sequences.
      *
+     * \details
```

Comment:
I thought everything after a new line is implicitly details?

Reply:
I thought so as well, but a look at the html output shows that the title line below is not parsed correctly if we omit the \details keyword.